bundler/helpers/v1/monkey_patches/definition_ruby_version_patch.rb

```diff
@@ -12,7 +12,7 @@ def index
           Gem::Specification.new("ruby\0", requested_version)
       end
 
-      %w(2.5.3p105 2.6.10p210 2.7.6p219 3.0.4p208).each do |version|
+      %w(2.5.3p105 2.6.10p210 2.7.6p219 3.0.4p208 3.1.4p223).each do |version|
```

Comment:
@deivid-rodriguez I've included the _current_ Ruby in this (Bundler v1) list. There's a more recent release (Ruby 3.1.5), but we're not running it yet. Does this list seem reasonable to you?

Reply:
I'm not sure if the patch part (`p105` for 2.5.3, etc) is needed or not. I suspect it was some Bundler 1 thing that made that necessary so just in case I'd keep them. Unfortunately I don't know of a good way of figuring out the patchlevel other than installing each ruby and checking the value of `RUBY_PATCHLEVEL`.

Reply:
We've had this out for about four hours now without issues. I'm going to merge it. I'll have to do a followup PR to add the patch part 😄 Thanks for all your input 🙇

Reply:
I've opened https://github.com/dependabot/dependabot-core/pull/9645 to add the patch levels 🙂